dotnet/src/Connectors/Connectors.HuggingFace/Core/HuggingFaceOpenAIClient.cs

```diff
@@ -0,0 +1,232 @@
+﻿// Copyright (c) Microsoft. All rights reserved.
+
+using System;
+using System.Collections.Generic;
+using System.Diagnostics.Metrics;
+using System.IO;
+using System.Linq;
+using System.Runtime.CompilerServices;
+using System.Text;
+using System.Threading;
+using System.Threading.Tasks;
+using Microsoft.Extensions.Logging;
+using Microsoft.SemanticKernel.ChatCompletion;
+using Microsoft.SemanticKernel.Connectors.HuggingFace.Client.Models;
+using Microsoft.SemanticKernel.Http;
+using Microsoft.SemanticKernel.Text;
+
+namespace Microsoft.SemanticKernel.Connectors.HuggingFace.Client;
+
+internal sealed class HuggingFaceOpenAIClient
+{
+    private readonly HuggingFaceClient _clientCore;
```

Comment:
This is internal so it doesn't matter much but I'm not fan of interdependencies. HuggingFaceOpenAIClient uses HuggingFaceClient object and static members, and HuggingFaceClient uses HuggingFaceOpenAIClient objec.

Reply:
Thanks, I didn't realize at first, you are right here, will correct the flow of dependency.